proxies/venue-graphql-proxy/src/schema/resolvers.ts

```diff
@@ -0,0 +1,14 @@
+import Venue from '../resolvers/Venue';
+import venueQueryResolver from '../resolvers/venueQueryResolver';
+import venuesByIdsResolver from '../resolvers/venuesByIdsResolver';
+
+const resolvers = {
+  Query: {
+    venue: venueQueryResolver,
+    venuesByIds: venuesByIdsResolver,
+  },
+  Venue,
+  // EventsConnection: LinkedPaginatedConnectionResolver,
+  // Event,
```

Comment:
Were this supposed to be removed while the whole Events datasource would be removed?

Reply:
done